VARIABLES.md

```diff
@@ -958,15 +1006,15 @@ Default:  "{{ monitoring_interceptors_enabled }}"
 
 ### control_center_user
 
-Only use to customize Linux User Control Center Service runs with. User must exist on host.
+Only use to customize Linux User Control Center Service runs with. Default user is cp-control-center
```

Comment:
Modify this variable to customize the Linux User that the Control Center Service runs with. Default user is cp-control-center.